modules/cert-manager-certificate-api-server.adoc

```diff
@@ -0,0 +1,70 @@
+// Module included in the following assemblies:
+//
+// * security/cert_manager_operator/cert-manager-creating-certificate.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="cert-manager-certificate-api-server_{context}"]
+= Creating certificates for API server
+
+.Prerequisites
+
+* You have access to the cluster with `cluster-admin` privileges.
+* You have installed {cert-manager-operator} 1.13.0 or above.
+
+.Procedure
+
+. Create an issuer. For more information, see "Configuring an issuer" in "Additional Resources" section.
+
+. Create a certificate:
+
+.. Create a YAML file, for example, `certificate.yaml`, that defines the `Certificate` object:
++
+.Example `certificate.yaml` file
++
+[source, yaml]
+----
+apiVersion: cert-manager.io/v1
+kind: Certificate
+metadata:
+  name: <tls_cert> #<1>
+  namespace: openshift-config
+spec:
+  isCA: false
+  commonName: "api.<cluster_base_domain>" #<2>
+  secretName: <tls_cert> #<3>
+  dnsNames:
+  - "api.<cluster_base_domain>" #<4>
+  issuerRef:
+    name: <issuer_name> #<5>
+    kind: Issuer
+----
+<1> Provide a name for the certificate.
+<2> Specify the common name (CN).
+<3> Specify the name of the secret to create that contains the certificate.
+<4> Specify the DNS name of the API server.
+<5> Specify the name of the issuer.
+
+.. Create the `Certificate` object by running the following command:
++
+[source, terminal]
+----
+$ oc create -f certificate.yaml
+----
+
+. Add the API server named certificate. For more information, see "Adding an API server named certificate" section in "Additional resources".
```

Comment:
This might make this a little more consistent with your first step:
```suggestion
. Add the API server named certificate. For more information, see "Adding an API server named certificate" section in the "Additional resources" section.
```